package.json

```diff
@@ -43,5 +43,10 @@
     "style-loader": "^0.16.0",
     "webpack": "^2.3.2",
     "webpack-dev-server": "^2.4.2"
+  },
+  "devDependencies": {
```

Comment:
I don't think these are `devDependencies`... that said, the whole project's dependencies need fixing 😕